src/platform/plugins/shared/dashboard/common/bwc/types.ts

```diff
@@ -9,7 +9,7 @@
 
 import type { SavedObjectReference } from '@kbn/core/public';
 import type { Serializable } from '@kbn/utility-types';
-import type { GridData } from '../../server/dashboard_saved_object';
+import { GridData } from '../content_management/v1/types'; // TODO: This will be moved in https://github.com/elastic/kibana/issues/221295
```

Comment:
I wonder if we need these TODO comments in here? If we need to call these out, should we do that in the linked issue?

Reply:
This comment is a reminder for myself - I will be tackling that issue right after this one, so I would rather keep the `TODO` around if that's okay?